### DISCUSSION TITLE
Scrolling problem in kitty

### DISCUSSION THREAD

**Discussion by certik** [Textualize/textual#2438] [Q&A]
I use kitty on macOS and I tried Frogmouth (as in https://github.com/Textualize/textual/issues/2435). When I scroll using the trackpad, the scrolling it "jerky" and sometimes gets stuck. Note that the scrolling with the default terminal app on macOS is very smooth.

(The rendering itself is very good in kitty, but quite bad in the default terminal.)

**Comment by davep** [comment]
It's possible that this could be down to something in your kitty configuration. I've been using kitty as my daily driver since just before starting work on Frogmouth, and haven't had any issues at all. You can find my configuration [over here](https://github.com/davep/kitty).

Meanwhile, are you able to share your configuration, and perhaps point to any markdown files you were viewing that are displaying this issue?

**Comment by willmcgugan** [comment]
I can reproduce this. It is a problem with Kitty. We should report it up-stream. The last time I looked, it appeared to be an error when Kitty was translating gestures in to mouse up/down events.
